check-before-deployment.md

```diff
@@ -0,0 +1,352 @@
+
+Take the `/dev/nvme0n1` data disk as an example:
+
+1. View the data disk.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    fdisk -l
+    ```
+
+    ```
+    Disk /dev/nvme0n1: 1000 GB
+    ```
+
+2. Create the partition table.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    parted -s -a optimal /dev/nvme0n1 mklabel gpt -- mkpart primary ext4 1 -1
+    ```
+
+    > **Note:**
+    >
+    > Use the `lsblk` command to view the device number of the partition: for a NVMe disk, the generated device number is usually `nvme0n1p1`; for a regular disk (for example, `/dev/sdb`), the generated device number is usually `sdb1`.
+
+3. Format the data disk to the ext4 filesystem.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    mkfs.ext4 /dev/nvme0n1p1
+    ```
+
+4. View the partition UUID of the data disk.
+
+    In this example, the UUID of nvme0n1p1 is `c51eb23b-195c-4061-92a9-3fad812cc12f`.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    lsblk -f
+    ```
+
+    ```
+    NAME    FSTYPE LABEL UUID                                 MOUNTPOINT
+    sda
+    ├─sda1  ext4         237b634b-a565-477b-8371-6dff0c41f5ab /boot
+    ├─sda2  swap         f414c5c0-f823-4bb1-8fdf-e531173a72ed
+    └─sda3  ext4         547909c1-398d-4696-94c6-03e43e317b60 /
+    sr0
+    nvme0n1
+    └─nvme0n1p1 ext4         c51eb23b-195c-4061-92a9-3fad812cc12f
+    ```
+
+5. Edit the `/etc/fstab` file and add the `nodelalloc` mount options.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    vi /etc/fstab
+    ```
+
+    ```
+    UUID=c51eb23b-195c-4061-92a9-3fad812cc12f /data1 ext4 defaults,nodelalloc,noatime 0 2
+    ```
+
+6. Mount the data disk.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    mkdir /data1 && \
+    mount -a
+    ```
+
+7. Check using the following command.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    mount -t ext4
+    ```
+
+    ```
+    /dev/nvme0n1p1 on /data1 type ext4 (rw,noatime,nodelalloc,data=ordered)
+    ```
+
+    If the filesystem is ext4 and `nodelalloc` is included in the mount options, you have successfully mount the data disk ext4 filesystem with options on the target machines.
+
+## Check and disable system swap
+
+This section describes how to disable swap.
+
+TiDB requires sufficient memory space for operation. It is not recommended to use swap as a buffer for insufficient memory, which might reduce performance. Therefore, it is recommended to disable the system swap permanently.
+
+Do not disable the system swap by executing `swapoff -a`, or this setting will be invalid after the machine is restarted.
+
+To disable the system swap, execute the following command:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+echo "vm.swappiness = 0">> /etc/sysctl.conf 
+swapoff -a && swapon -a
+sysctl -p
+```
+
+## Check and stop the firewall service of deployment machines
```

Comment:
```suggestion
## Check and stop the firewall service of target machines
```
In docs-cn repo, @kissmydb once suggested that we unify the terms and use "中控机" and "目标机器"  (instead of "部署机"). So in docs repo, I suggest we use "Control Machine" and "Target Machine" (instead of "deployment machine") across our documents. @ran-huang Please update all places that apply to this suggestion.

Reply:
Updated.